command-acceptor/app/controller/interfaces/RequestAnalyser.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from app.models.ErrorMessage import ErrorMessage
+from app.models.AcceptedMessage import AcceptedMessage
+
+
+class RequestAnalyser(ABC):
```

Comment:
Analyser has to declare some action as interface
i.e. RequestAnalyser -> analyse and result could be -> Message or thrown exception with ErrorMessage

Reply:
Done. But not sure about implementation. I am returning message as a result of process method. Maybe message should be available by .message method